mls-rs/src/group/message_processor.rs

```diff
@@ -468,9 +465,10 @@ pub(crate) enum EventOrContent<E> {
     all(not(target_arch = "wasm32"), mls_build_async),
     maybe_async::must_be_async
 )]
-pub(crate) trait MessageProcessor: Send + Sync {
+pub(crate) trait MessageProcessor: Send + Sync + Sized {
     type OutputType: TryFrom<ApplicationMessageDescription, Error = MlsError>
         + From<CommitMessageDescription>
+        // From<CommitProcessor>
```

Comment:
delete?

Reply:
yeah, in the follow up which depends on this one